sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/dashboard/contacts/ContactsDashboardSteps.java

```diff
@@ -0,0 +1,48 @@
+package org.sormas.e2etests.steps.web.application.dashboard.contacts;
+
+import cucumber.api.java8.En;
+import javax.inject.Inject;
+import org.assertj.core.api.SoftAssertions;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pages.application.dashboard.Surveillance.ContactsDashboardPage;
+
+public class ContactsDashboardSteps implements En {
+
+  private final WebDriverHelpers webDriverHelpers;
+  private final SoftAssertions softly;
+  private Integer covid19ContactsCounterIntValueAfter;
```

Comment:
remove intValue from name, and change Integer to its primitive

Reply:
Good point. I will change it.